server/src/db/migrations/20231101154655_createReviews.cjs

```diff
@@ -0,0 +1,36 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("reviews", (table) => {
+        table.bigIncrements("id")
+
+        table.bigInteger("userId")
+            .notNullable()
+            .index()
+            .unsigned()
+            .references("users.id")
+        table.bigInteger("destinationId")
+            .notNullable()
+            .index()
+            .unsigned()
+            .references("destinations.id")
+        table.integer("rating").notNullable()
+        table.string("title")
+        table.text("reviewContent")
```

Comment:
ideally, we don't repeat the table name for a column. This should be called `content`, so that way we can access as `review.content` rather than `review.reviewContent`